driver-core/src/test/functional/com/mongodb/operation/MixedBulkWriteOperationSpecification.groovy

```diff
@@ -490,6 +490,47 @@ class MixedBulkWriteOperationSpecification extends OperationFunctionalSpecificat
         async << [true, false]
     }
 
+    @Category(Slow)
```

Comment:
These tests are pretty fragile as they depend on the length of the collection name and maxMessageSize being equal to 48,000,000.